<launch>
<!-- Launch file for Mapping -->

	<arg name="rviz" default="true" />

	<!-- <param name="config_file" type="string" value="/config/avia.yaml" /> -->
	<!-- <param name="config_file" type="string" value="/config/mid360.yaml" /> -->
	<!-- <param name="config_file" type="string" value="/config/horizon.yaml" /> -->
	<!-- <param name="config_file" type="string" value="/config/velodyne.yaml" /> -->
	<!-- <param name="config_file" type="string" value="/config/ouster64.yaml" /> -->

	<!-- <param name="config_file" type="string" value="/config/dataset/nclt.yaml" /> -->

	<!-- <param name="config_file" type="string" value="/config/mapping_backup.yaml" /> -->
	<param name="config_file" type="string" value="/config/mapping_dev.yaml" />
	<!-- <param name="config_file" type="string" value="/config/mapping_imu_backshaft.yaml" /> -->
	<!-- <param name="config_file" type="string" value="/config/mapping_pointlio.yaml" /> -->
	<!-- <param name="config_file" type="string" value="/config/dataset/mapping_Hilti.yaml" /> -->
	<!-- <param name="config_file" type="string" value="/config/dataset/mapping_Hilti2.yaml" /> -->
	<!-- <param name="config_file" type="string" value="/config/dataset/mapping_UrbanLoco_HK.yaml" /> -->
	<!-- <param name="config_file" type="string" value="/config/dataset/mapping_UrbanLoco_CA.yaml" /> -->

	<param name="globalMapVisualizationSearchRadius" type="double" value="1000" />
	<param name="globalMapVisualizationPoseDensity" type="double" value="3" />
	<param name="globalMapVisualizationLeafSize" type="double" value="0.2" />

	<param name="map_update_mode" type="bool" value="false" />
    <node pkg="fast_lio_sam" type="fastlio_sam_ros1" name="system" output="screen" /> 

	<group if="$(arg rviz)">
		<node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find fast_lio_sam)/rviz_cfg/mapping.rviz" />
	</group>

</launch>
